addon/stories/pix-multi-select.stories.js

```diff
@@ -0,0 +1,122 @@
+import { hbs } from 'ember-cli-htmlbars';
+import centered from '@storybook/addon-centered/ember';
+
+export default { title: 'MultiSelect' };
```

Comment:
Suggestion:
Le mettre dans le dossier `Form` à côté de `Select`
```js
export default { title: 'Form' };
```

Reply:
à priori on ne peut pas mettre un double title identique . 

 Duplicate title 'Form' used in multiple files; . il faut que je creuse un peu storybook de ce côté là .